### PR TITLE
Add CMG (Checkmate Gaming)

### PR DESCRIPTION
**Pull request by coreystone**
About: Site used to host gaming tournaments and competitions.
Similarweb ranking: [#60,455](https://www.similarweb.com/website/checkmategaming.com/#overview)

### CODE REVIEW

File: entries/c/checkmategaming.com.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "CMG": {
+    "domain": "checkmategaming.com",
```
Comment (hkamran80) [inline entries/c/checkmategaming.com.json:3]:
```suggestion
    "domain": "checkmategaming.com",
    "img": "checkmategaming.com.png",
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by coreystone** [comment]
I apologize, I didn't resize the .png. What should I do?

**Comment by hkamran80** [comment]
You need to add an `img` tag to the JSON entry. You can click "Commit suggestion" on the requested changes above to add it automatically.

**Comment by coreystone** [comment]
> You need to add an `img` tag to the JSON entry. You can click "Commit suggestion" on the requested changes above to add it automatically.

Thanks for the quick and helpful response. I've committed the suggestion. This was my first pull request so still figuring out what I'm doing. Also forgot to resize the .png:  `Error:  PNGs must be one of the following sizes: 32x32, 64x64, 128x128.`

Resized to 128x128, tests are passing now.

**Comment by hkamran80** [comment]
Perfect, thanks! Glad to help! I'll finish reviewing it in a little while.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
It looks like you squished the PNG when resizing it.
Keeping the aspect ratio and simply adding padding makes the content more legible.

I suggest you use this image instead:

**Review by Carlgo11 — APPROVED** [review]

**Review by hkamran80 — APPROVED** [review]

**Comment by coreystone** [comment]
Thank you both for your help and for bearing with me. I'm excited to contribute more in the future 😄

**Comment by hkamran80** [comment]
Of course! Thanks for contributing!

**Pull request merged**